File: app/src/main/java/com/kelsos/mbrc/services/NowPlayingService.java
package com.kelsos.mbrc.services;

import com.kelsos.mbrc.data.NowPlaying;
import com.kelsos.mbrc.data.Page;

import rx.Observable;


public interface NowPlayingService {
  Observable<Page<NowPlaying>> getNowPlaying(int offset, int limit);
}

File: app/src/main/kotlin/com/kelsos/mbrc/services/NowPlayingService.kt
package com.kelsos.mbrc.services

import com.kelsos.mbrc.data.NowPlaying
import com.kelsos.mbrc.data.Page

import io.reactivex.Observable


interface NowPlayingService {
  fun getNowPlaying(offset: Int, limit: Int): Observable<Page<NowPlaying>>
}
